### ISSUE TITLE
[p]casino info causes exception if used whip Threshold is on. 

### ISSUE THREAD

**Issue by Omnipawn** [Redjumpman/Jumper-Cogs#66]
```py
Exception in command 'casino info'
Traceback (most recent call last):
  File "lib\discord\ext\commands\core.py", line 50, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "C:\Red-DiscordBot\cogs\casino.py", line 771, in _info_casino
    threshold = settings["Threshold"]
KeyError: 'Threshold'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "lib\discord\ext\commands\bot.py", line 846, in process_commands
    yield from command.invoke(ctx)
  File "lib\discord\ext\commands\core.py", line 634, in invoke
    yield from ctx.invoked_subcommand.invoke(ctx)
  File "lib\discord\ext\commands\core.py", line 374, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "lib\discord\ext\commands\core.py", line 54, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: KeyError: 'Threshold'

```

**Comment by Omnipawn** [comment]
Casino version: 1.7.08
Ran forceupdate. 
Toggling threshold off fixes issue, tried both with very small, default, and large thresholds.

**Comment by Redjumpman** [comment]
How long have you had casino installed on your bot? Are you a new user? Thank you for the detailed response, I'll be looking into this soon.

**Comment by Omnipawn** [comment]
I've been running it for a little over a month without trying the threshold function. As soon as I toggled it on for the first time, this exception started occurring

**Comment by Redjumpman** [comment]
Try the new update I just pushed. It should fix this issue from happening. I'll keep the issue open for a couple of days pending your response.

**Comment by Omnipawn** [comment]
The update has fixed the issue. I appreciate the speedy response!

**Comment by Redjumpman** [comment]
Awesome. Issue fixed via PR #65
